src/ofxSyphonClient.mm

```diff
@@ -60,6 +60,7 @@
     serverName = s.serverName;
 
     [pool drain];
+	return *this;
```

Comment:
Good catch but generally it's helpful to split PR's so each one deals with a particular issue

Reply:
Sure thing. Maybe I slip this other one in the same branch.
You can merge as it is or if you prefer I revert and make another PR later today or tomorrow.